app/views/application/_footer.html.haml

```diff
@@ -1,10 +1,15 @@
-%footer.footer
-  .col-sm-6
-    - unless current_user
-      %select#select-locale
-        = options_for_select linked_language_options, selected_language_option
+- if show_loomio_org_marketing
+  %footer.footer
+    .col-sm-6
+      - unless current_user
+        %select#select-locale
+          = options_for_select linked_language_options, selected_language_option
 
-    = link_to t(:translate, default: "help translate Loomio!"), 'https://www.loomio.org/d/hB1Oijwv/overview-of-the-translation-process-for-translators'
+      = link_to t(:translate, default: "help translate Loomio!"), 'https://www.loomio.org/d/hB1Oijwv/overview-of-the-translation-process-for-translators'
 
-  .col-sm-6.footer-links
-    = render "footer_links", target: 'blank'
+    .col-sm-6.footer-links
+      = render "footer_links", target: 'blank'
+- else
+  %footer.footer
+    .col-xs-12
+      = t :"footer.independent_install_html", hostname: ENV['CANONICAL_HOST']
```

Comment:
this should ideally be ideally abstracted / wrapped in a method like : 

```ruby 
= t :"footer.independent_install_html", hostname: canonical_hostname
``` 

so we can define some fallbacks somewhere and